### PR TITLE
Create blox.json

### PR DESCRIPTION
**Pull request by soudblox**

### CODE REVIEW

File: domains/blox.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "blox",
```
Comment (MaskDuck) [inline domains/blox.json:3]: gh username pls???

**Review by MaskDuck — APPROVED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
actually not approval

File: domains/blox.json (hunk 1 above)
Comment (MaskDuck) [reply domains/blox.json:3]: @soudblox

**Comment by soudblox** [comment]
updated

**Review by MaskDuck — APPROVED** [review]
sus alert

**Comment by soudblox** [comment]
> sus alert

what is lmao

**Pull request merged**